lib/galaxy/webapps/galaxy/controllers/workflow.py

```diff
@@ -740,7 +741,7 @@ def editor(self, trans, id=None, workflow_id=None, version=None):
         }
 
         # parse to mako
-        return trans.fill_template("workflow/editor.mako", editor_config=editor_config)
+        return editor_config
```

Comment:
Do you want to split out the mako elimination here ? None of this controller code should survive, we should already have all you need in the regular API.

Reply:
Yes, let's go step by step.